packages/platform-server/test/hydration_spec.ts

```diff
@@ -129,7 +129,7 @@ function verifyAllNodesClaimedForHydration(el: HTMLElement, exceptions: HTMLElem
     return;
   }
 
-  if (!(el as any).__claimed) {
+  if ((el as HydrationNode).__hydration?.status !== 'claimed') {
```

Comment:
We can use helper function mentioned above:
```suggestion
  if (readHydrationInfo(el)?.status !== 'claimed') {
```